tslint.json

```diff
@@ -1,122 +0,0 @@
-{
```

Comment:
Why did you get rid of this file?

Reply:
I was getting following error:
[tslint] interface name must start with a capitalized I (interface-name)
for code snippet:
interface DataNode {....}
i checked for solution online and all suggested to add 
"interface-name": false in the tslint.json file and this resolved my issue.
But i was not sure if this needs to be added in main code or was it just some local compilation issue

Reply:
Added the file back. By mistake i deleted the file